test/specs/languages/test-emacs-lisp.el

```diff
@@ -0,0 +1,57 @@
+;;; test-emacs-lisp.el --- Flycheck Specs: Haskell      -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2013-2016 Sebastian Wiesner and Flycheck contributors
+
+;; Author: Clément Pit-Claudel <clement.pitclaudel@live.com>
+
+;; This file is not part of GNU Emacs.
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; Specs for Emacs Lisp support.
+
+;;; Code:
+
+(require 'flycheck-buttercup)
+
+(defconst tel--excluded-checkdoc-vars
```

Comment:
I don't think that we need a special prefix for each test file.  My other tests just use `flycheck/`, i.e. a trailing slash instead of dash to avoid conflicts with Flycheck itself.  That should be sufficient, and it's easier to read imho.  It took me a while to understand what `tel` actually stands for :blush: